app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -331,6 +331,16 @@ controlPlane:
   # -- If true, then control plane can support TLS secrets for builtin gateway outside of mesh system namespace.
   # The downside is that control plane requires permission to read Secrets in all namespaces.
   supportGatewaySecretsInAllNamespaces: false
+  # -- DNS configuration for the pod.
+  dns:
+    # -- Defines how DNS resolution is configured for that Pod.
```

Comment:
should we state that this is equivalent to https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy and what values are available?